src/app/transactions/MempoolFeePieChart.tsx

```diff
@@ -0,0 +1,139 @@
+import { useColorModeValue } from '@chakra-ui/react';
+import { Cell, Pie, PieChart } from 'recharts';
+
+const pieChartWidth = 215;
+const pieChartHeight = 200;
+
+const renderCustomizedLabel = ({
+  percent,
+  cx,
+  cy,
+  midAngle,
+  outerRadius,
+}: {
+  percent: number;
+  x: number;
+  y: number;
+  cx: number;
+  cy: number;
+  midAngle: number;
+  outerRadius: number;
+}) => {
+  if (!percent) return null; // Don't show zeros
+  const RADIAN = Math.PI / 180;
+  const radius = outerRadius + 25; // Increase radius to move label further out
+  const newX = cx + radius * Math.cos(-midAngle * RADIAN);
+  const newY = cy + radius * Math.sin(-midAngle * RADIAN);
+  return (
+    <text
+      x={newX}
+      y={newY}
+      fill="#737680"
+      fontSize="12px"
```

Comment:
same for color and styles here

Reply:
These svg elements do not have access to chakra ui theme information, which is why colors are passed in and I am setting sizes manually